github/ci/prow-deploy/files/jobs/kubevirt/kubevirt.github.io/kubevirt-github-io-postsubmits.yaml

```diff
@@ -0,0 +1,42 @@
+postsubmits:
+  kubevirt/kubevirt.github.io:
+    - name: publish-img
+      always_run: false
+      run_if_changed: "Makefile"
+      annotations:
+        testgrid-create-test-group: "false"
+      decorate: true
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-kubevirtci-quay-credential: "true"
+      cluster: phx-prow
+      spec:
+        nodeSelector:
+          type: vm
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/bash"
+              - "-c"
+              - |
+                  cat $QUAY_PASSWORD | docker login --username $(<$QUAY_USER) --password-stdin quay.io && \
+                  GITHUB_USER=kubevirt-bot GITHUB_TOKEN=$(cat /etc/github/oauth) && \
```

Comment:
What do you need the github token for? If you need it you might consider using GIT_ASKPASS so you do not accidentally leak the token: https://github.com/kubevirt/project-infra/blob/b40949a970182e1a16986d66161f272c3e00e27c/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml#L247

More info about GIT_ASKPASS: https://git-scm.com/docs/gitcredentials

Reply:
I have no idea what this means.  I copy/pasted from another job.

Reply:
I mean that you can just use the git credential mechanism by using the GIT_ASKPASS env var as here: https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml#L102-L103

Then you also need to add an extra_ref which will check out the project-infra repository that makes the script you are pointing to via the above variable available: https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml#L81-L83

And then you can remove the catting of the oauth token file (which will then be done by GIT_ASKPASS under the hood).

 ```suggestion
```